lib/surface_site_web/live/components_basics.ex

```diff
@@ -75,6 +75,10 @@ defmodule SurfaceSiteWeb.ComponentsBasics do
                 This way you can remove the implemented `render/1` altogether. Any defined assign or function
                 will be available in the new template.
 
+                > **Note:** Unlike EEX templates which you can use with any kind of text file,
+                Surface extends HTML. For this reason colocated templates are not named with an `*.html.sface` suffix
+                but with the suffix `*.sface`.
```

Comment:
Good call adding this note! 👍

Do you think the following sounds more fluid?

```suggestion
                > **Note:** Unlike EEX templates, which can be used with any kind of text file, Surface's templates 
                already extend HTML. Therefore, they are named simply as `*.sface` instead of `*.html.sface`.
```